xarray/tests/duckarrays/test_sparse.py

```diff
@@ -33,6 +23,9 @@ def create(op, shape, dtypes):
     def convert(arr):
         if arr.ndim == 0:
             return arr
+        # sparse doesn't support float16
+        if np.issubdtype(arr.dtype, np.float16):
+            return arr
```

Comment:
Is there a better way? `pytest.skip` seems to skip absolutely everything

Reply:
we can make sure to never even call `convert` by filtering out the dtypes we would like to ignore.